.templates/template_location_v2.json

```diff
@@ -178,5 +178,13 @@
     "templateLanguages": ["python"],
     "runPlatforms": ["cloudfunctions"],
     "entryPoint": "hello_get"
+  },
+  {
+    "repoPath": "https://github.com/GoogleCloudPlatform/cloud-code-samples.git",
+    "directoryPath": "java/java-dataflow-samples/read-pubsub-write-bigquery",
+    "templatePath": "read-pubsub-write-bigquery",
```

Comment:
nit: I see that `read-pubsub-write-bigquery` is repeated, in the `directoryPath` and `templatePath`. Just wanted to make sure this naming was intentional, since the other examples have different naming between the fields

Reply:
this is only used in metrics as a template type.
for CC samples we have "hello-world" and "guestbook"
for dataflow you can pick any meaningful name here